pom.xml

```diff
@@ -52,19 +52,18 @@
     </issueManagement>
 
     <properties>
-        <version.arquillian_core>1.1.11.Final</version.arquillian_core>
-        <version.arquillian_cube>1.0.0.Alpha14</version.arquillian_cube>
+        <version.arquillian_core>1.1.12.Final</version.arquillian_core>
+        <version.arquillian_cube>1.0.0.Alpha20</version.arquillian_cube>
         <version.toxiproxyjava>2.1</version.toxiproxyjava>
-        <version.junit>4.11</version.junit>
-        <version.mockito>1.10.19</version.mockito>
-        <version.assertj>2.4.1</version.assertj>
-        <version.arquillian.recorder>1.1.5.Final</version.arquillian.recorder>
+        <version.junit>4.12</version.junit>
+        <version.mockito>2.7.17</version.mockito>
+        <version.assertj>3.6.2</version.assertj>
+        <version.arquillian.reporter>0.0.2</version.arquillian.reporter>
```

Comment:
please add to this commit only relevant changes - updates of junit or arquillian-core versions put into a separate commit